### PR TITLE
Fix applying of EBS Volume tags

### PR DESCRIPTION
**Pull request by sanyer**
fixes https://github.com/mitchellh/packer/issues/3573 and provides some extra minor enhancements

### CODE REVIEW

**Comment by sanyer** [comment]
travis fails on go 1.8beta1 during github.com/mitchellh/packer/packer/rpc installation, kinda weird

**Comment by mwhooker** [comment]
awesome, thanks!

File: builder/amazon/ebs/step_tag_ebs_volumes.go [hunk 1]
```diff
@@ -17,37 +18,35 @@ func (s *stepTagEBSVolumes) Run(state multistep.StateBag) multistep.StepAction {
...
-			state.Put("error", err)
-			ui.Error(err.Error())
-			return multistep.ActionHalt
-		}
+	ui.Say(fmt.Sprintf("Adding tags to source EBS Volumes:"))
+	tags := common.ConvertToEC2Tags(s.VolumeRunTags, ui)
+
+	_, err := ec2conn.CreateTags(&ec2.CreateTagsInput{
+		Resources: []*string{
+			instance.BlockDeviceMappings[0].Ebs.VolumeId,
```
Comment (mwhooker) [inline builder/amazon/ebs/step_tag_ebs_volumes.go]: I know this is existing behavior, but shouldn't we create a tag for each volume in the volumeIds list?

**Comment by mwhooker** [comment]
thanks! going to merge in #4420

File: builder/amazon/ebs/step_tag_ebs_volumes.go (hunk 1 above)
Comment (sanyer) [inline builder/amazon/ebs/step_tag_ebs_volumes.go]: that's a good point :-)
especially considering the fact that volumeIds was populated with list of BlockDeviceMappings...

Comment (sanyer) [inline builder/amazon/ebs/step_tag_ebs_volumes.go]: tracked it in #4428

**Pull request closed without merging**